content_schemas/formats/shared/definitions/_logo.jsonnet

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Shall we call this file organisation logo? I'm wondering if logo is a bit too generic as it's definitely constrained to a few scenarios.

Reply:
Good idea, I'll rename this.